docker-for-mac/faqs.md

```diff
@@ -220,130 +131,40 @@ export DOCKER_HOST=unix:///var/run/docker.sock
 
 ### How do I connect from a container to a service on the host?
 
-The Mac has a changing IP address (or none if you have no network access). Our
-current recommendation is to attach an unused IP to the `lo0` interface on the
+Mac has a changing IP address (or none if you have no network access). We recommend that you attach an unused IP to the `lo0` interface on the
```

Comment:
Is part of this sentence missing?

Reply:
No, it is not. Git only displays the delta. Here's the full info - https://deploy-preview-9596--docsdocker.netlify.com/docker-for-mac/faqs/#how-do-i-connect-from-a-container-to-a-service-on-the-host